src/resolvers/userResolver.js

```diff
@@ -83,6 +92,7 @@ const userResolver = {
         };
       }
 
+      req.session.userId = user.id;
```

Comment:
Was this meant to be an if condition?

Reply:
Here, the `userId` of the user that just logged in is being saved to our session. The same is being done in the `register` mutation.